### ISSUE TITLE
Will the elasticsearch client work in a ARM64 platform like the coral devboard?

### ISSUE THREAD

**Issue by RBAM** [elastic/elasticsearch-py#1302]
Hi

I am using the Coral devboard (a ARM64 platform).
My project consists of taking a photo and doing inference (AI, image classification) with the devboard. Then I want to send the results directly to Elasticsearch.
Currently I am using a PC to index the devboards results. Check the image.
Will there be a problem? I read this: Support-for-arm64-raspberry-pi-4. So I am not sure.
I only want to send the results (a dictionary/json) and probably the image with the devboard; as I already have a server to visualize with Kibana or search with Elasticsearch.

Anyway, if it is not possible it would be nice to have it in the future.

Thank you

![image](https://user-images.githubusercontent.com/12815073/86034587-e7390580-ba08-11ea-8708-7fd845089c15.png)

**Comment by sethmlarson** [comment]
The Elasticsearch Python client is pure-Python and releases non-binary wheels so you shouldn't have any issue installing and using the client on any CPU architecture you've installed Python on. Let me know if you have additional questions.

**Comment by Namburger** [comment]
oh nice! 
```
python3 -m pip install elasticsearch
```

**Comment by RBAM** [comment]
@Namburger Have you tested on the board? did it work?

**Comment by Namburger** [comment]
Well the client certainly works but  I have not stand up a server to test

**Comment by RBAM** [comment]
@Namburger 
Hello

I have installed it in the devboard. 

I wrote this code to test connection from the devboard to the laptop (Windows) which has Elasticsearch installed.
But I was not successful.

import requests
res = requests.get('http://172.16.1.132:9200')
print(res.content)

![image](https://user-images.githubusercontent.com/12815073/86945596-37a21880-c117-11ea-9129-85e30ca67367.png)




I also tested this: 
![image](https://user-images.githubusercontent.com/12815073/86945439-045f8980-c117-11ea-9c8f-b0d49d79625d.png)


Any idea on how can I test the communication?
I put the firewall down and only then I can ping the windows PC. 
I also opened port 9200 in the PC. But still I cannot communicate.
